proposals/0189-Restructuring-OnResetTimeout.md

```diff
@@ -106,12 +104,13 @@ _OnResetTimeout_ function definition would be as follows for the _BasicCommunica
 	</param>
 </function>
 ```
+Since HMI API does not support(as i have been told) _deprecated_, we are going to remove the older implementation.
```

Comment:
Update to:

Since HMI API does not support _deprecated_, it is going to be removed from the older implementations.

Reply:
Done